test/test_site/testTagDivs.md

```diff
@@ -0,0 +1,24 @@
+# Div with shown tag
+<div tags="tag--shown">
+Div with shown tag
+</div>
+
+# Div with shown-2 tag
+<div tags="tag--shown-2">
+Div with shown-2 tag
+</div>
+
+# Div with multiple tags
+<div tags="tag--shown tag--other">
+Div with multiple tags
+</div>
+
+# Div with shown on overwrite tag
```

Comment:
Override is the correct term, as used in **Page.js** [above](https://github.com/MarkBind/markbind/pull/467/files#diff-6e6e830d57ea503d762ce14927385fc6R426):

> ```js
> // Included tags specified in front matter will override included tags specified in site.json
> ```

See: https://stackoverflow.com/questions/8651562/overwrite-or-override